cmd/ftp-server-driver.go

```diff
@@ -82,7 +82,10 @@ func (m *minioFileInfo) Mode() os.FileMode {
 }
 
 func (m *minioFileInfo) ModTime() time.Time {
-	return m.info.LastModified
+	if !m.info.LastModified.IsZero() {
+		return m.info.LastModified
+	}
+	return time.Now().UTC()
```

Comment:
Using a fixed timestamp would yield in consistent results during different runs. I would prefer to use 1-1-1970 (epoch), but MS-DOS timestamps only supports dates from 1-1-1980 ([source](https://learn.microsoft.com/en-us/windows/win32/api/winbase/nf-winbase-filetimetodosdatetime#remarks)), so it may be safer to use that instead.
```suggestion
        return minFileDate;
```
Where `minFileDate` is defined as
```golang
var minFileData = time.Date(1980, 1, 1, 0, 0, 0, 0, time.UTC)    // Workaround for Filezilla
```